x-pack/plugins/reporting_export_type_plugin/server/plugin.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { PluginInitializerContext, CoreSetup, CoreStart, Plugin, Logger } from '@kbn/core/server';
+
+import { ConfigSchema } from '@kbn/reporting-plugin/server/config';
+import { ExportTypesPluginSetup, ExportTypesPluginStart } from './types';
+
+export class ExportTypesPlugin
+  implements Plugin<{}, {}, ExportTypesPluginSetup, ExportTypesPluginStart>
+{
+  exportTypes = [
+    // new CsvExportType(),
+    // new PdfExportType(),
+    // new PngExportType(),
```

Comment:
Not important for now, but we'll probably want to `new` these objects in the `constructor`, so we can pass in the logger and reporting config.

Reply:
Speaking of config, another thing we'll have to add to the ReportingSetup/ReportingStart API will be method to get the config from Reporting.